samples/Compute/allowed-platform-images/azurepolicy.json

```diff
@@ -1,10 +1,18 @@
 {
```

Comment:
Please test this in the portal before submitting the next iteration. 

Happy to help if you have trouble with any of the error messages :)